File: Lib/Telerik/Examples/QuickStart/WordsProcessing/WordMailMerge/SampleData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Telerik.Examples.WinControls.WordsProcessing.WordMailMerge
{ 
    public class Person
    {
        private readonly string firstName;
        private readonly string lastName;

        public Person(string firstName, string lastName)
        {
            this.firstName = firstName;
            this.lastName = lastName;
        }

        public string FirstName
        {
            get
            {
                return this.firstName;
            }
        }

        public string LastName
        {
            get
            {
                return this.lastName;
            }
        }

        public static IEnumerable<Person> Persons
        {
            get
            {
                return new Person[] {
                    new Person("Andrew", "Fuller"),
                    new Person("Nancy", "Davolio"),
                    new Person("Janet", "Leverling"),
                    new Person("Margaret", "Peacock"),
                    new Person("Robert", "King"),
                    new Person("Horst", "Kloss"),
                };
            }
        }
    }
}

File: Lib/Telerik/Examples/QuickStart/WordsProcessing/WordMailMerge/SampleData.vb
Imports System.Text

Namespace Telerik.Examples.WinControls.WordsProcessing.WordMailMerge
    Public Class Person
        'INSTANT VB NOTE: The variable firstName was renamed since Visual Basic does not allow class members with the same name:
        Private ReadOnly firstName_Renamed As String
        'INSTANT VB NOTE: The variable lastName was renamed since Visual Basic does not allow class members with the same name:
        Private ReadOnly lastName_Renamed As String

        Public Sub New(ByVal firstName As String, ByVal lastName As String)
            Me.firstName_Renamed = firstName
            Me.lastName_Renamed = lastName
        End Sub

        Public ReadOnly Property FirstName() As String
            Get
                Return Me.firstName_Renamed
            End Get
        End Property

        Public ReadOnly Property LastName() As String
            Get
                Return Me.lastName_Renamed
            End Get
        End Property

        Public Shared ReadOnly Property Persons() As IEnumerable(Of Person)
            Get
                Return New Person() {New Person("Andrew", "Fuller"), New Person("Nancy", "Davolio"), New Person("Janet", "Leverling"), New Person("Margaret", "Peacock"), New Person("Robert", "King"), New Person("Horst", "Kloss")}
            End Get
        End Property
    End Class
End Namespace
